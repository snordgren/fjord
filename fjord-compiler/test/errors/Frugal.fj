module errors.Frugal

record Message
  content : String


frugal : Message -* (Message, Message)
frugal m =
  (m, m)

