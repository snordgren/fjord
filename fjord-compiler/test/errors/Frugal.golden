errors/Frugal.fj:9:7:
  |
9 |   (m, m)
  |       ^^^
too many usages of m
